app/views/unattended/report_templates/ansible_-_ansible_inventory.erb

```diff
@@ -158,7 +158,7 @@ require:
       inventory_data.update('location': host.location) if input('Location') == 'yes'
       hostgroup = host.hostgroup.title if host.hostgroup
       inventory_data.update('host_group': hostgroup) if input('Host Group') == 'yes'
-      host_collections = host.host_collections.map { |c| c.name } if host.host_collections
+      host_collections = host.host_collections.map { |c| c.name } if input('Host Collections') == 'yes' and host.host_collections
```

Comment:
Can we here use `&&`? I think we force this to be used everywhere in the code. Not a blocker though.

Reply:
Oh yeah, I always forget that in Ruby land &&  is preferred. Been staring at Python for too long.